nova/engines/btdb.py

```diff
@@ -26,7 +26,13 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
-from HTMLParser import HTMLParser
+try:
```

Comment:
What about this style? The intention is a bit more clear IMO:
```python
import sys
if (sys.version_info > (3, 0)):
    # Python 3 imports in this block
else:
    # Python 2 imports in this block
```

Reply:
Also you can merge after you changed it (or not), I believe you've made sure it is compatiable to both py versions.

Reply:
@Chocobo1 ~~I like the idea but I prefer `if sys.version_info[0] >= 3:` and remove the #python3/2 comments.
I opened several PR, if you want, after everything is merged you can open a new PR with this change in all plugins and close this PR.~~

I don't know if all libraries are available from python  3.0. There are a lof of changes between python 3.0 and python 3.7...

Reply:
>I don't know if all libraries are available from python 3.0. There are a lof of changes between python 3.0 and python 3.7...


If you have time, you could trigger TravisCI to test it, it does test it with python 3.7 nightly.